src/uds/ecu_reset.rs

```diff
@@ -60,8 +59,11 @@ impl UdsDiagnosticServer {
     ///
     /// ## Parameters
     /// * server - The UDS Diagnostic server
-    pub fn disable_rapid_power_shutdown(&mut self) -> DiagServerResult<()> {
-        self.execute_command_with_response(ECUReset, &[ResetType::DisableRapidPowerShutDown.into()])
-            .map(|_| ())
+    pub fn uds_disable_rapid_power_shutdown(&mut self) -> DiagServerResult<()> {
+        self.send_command_with_response(
+            UdsCommand::ECUReset,
+            &[ResetType::DisableRapidPowerShutDown.into()],
+        )
+        .map(|_| ())
```

Comment:
Do you think this form is a bit more readable? Mapping takes a bit more mental strain :)  (same for several other places above)

```suggestion
        )?;
        Ok(())
```